.travis.yml

```diff
@@ -3,24 +3,24 @@ language: go
 dist: xenial
 
 go:
-- 1.13.4
+  - 1.13.4
 
 before_script:
   - if [ "$DOCKER_USERNAME" ]; then docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"; fi
   - ./install-operator-sdk.sh
-  - git clone --depth 1 -b "v0.6.0" --single-branch https://github.com/k8s-school/kind-travis-ci.git
-  - ./kind-travis-ci/kind/k8s-create.sh -s
+  - git clone --depth 1 -b "master" --single-branch https://github.com/k8s-school/kind-travis-ci.git
```

Comment:
Maybe using a tag here would be more robusts, please use `git tag` on kind-travis-ci repository to pick latest one.

Reply:
I'll use the [`k8s-v1.18.2`](https://github.com/k8s-school/kind-travis-ci/commits/k8s-v1.18.2) tag then?